Sources/PaymentsCore/PaymentsCoreConstants.swift

```diff
@@ -0,0 +1,6 @@
+/// :nodoc: This class is exposed for internal PayPal use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+public class PayPalCoreConstants {
+    
+    /// :nodoc: This property is exposed for internal PayPal use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    public static let paypalSDKVersion: String = "0.0.3"
```

Comment:
I see in some places we use `paypal` and others `payPal`. We should probably decide on a format and update the rest of the SDK to reflect our preference. Not something for this PR but should we make a follow up ticket for that work?

Reply:
DONE! `DTNOR-625` is in the Backlog and references this convo ✅

Reply:
Good catch., I _thiiiink_ the standard is `payPal`